graylog2-web-interface/src/components/common/InputDescription.jsx

```diff
@@ -0,0 +1,56 @@
+// @flow strict
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import styled, { type StyledComponent } from 'styled-components';
+
+import type { ThemeInterface } from 'theme';
+import { HelpBlock } from 'components/graylog';
+
+const Wrapper: StyledComponent<{hasError: boolean}, ThemeInterface, HTMLSpanElement> = styled.span(({ theme, hasError }) => `
+  ${hasError ? `
+    color: ${theme.colors.variant.danger};
+  ` : ''};
+`);
```

Comment:
I made sure the text color changes when there is an error, this way the component can be used independently of the  `Input` component. I tried to style the `HelpBlock`, but it is not possible because the `hasError` attribute would be added to the DOM.